schemas/controlled_vocabulary_schemas/EGA.cv.assay_subtype_by_sequencer.json

```diff
@@ -0,0 +1,10 @@
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "type": "string",
+    "$id": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/controlled_vocabulary_schemas/EGA.cv.assay_subtype_by_sequencer.json",
+    "title": "EGA Controlled Vocabulary (CV) for assay subtypes by sequencer [EFO:0003740]",
+    "meta:version": "0.0.0",
+    "description": "Controlled Vocabulary (CV) list for assay subtypes by sequencer: any ontologized term for a subtype (i.e. child ontology) of an sequencing assay [EFO:0003740].",
+    "enum": [ "bulk immune repertoire sequencing", "HTGTS-Rep-seq", "Ig-Seq", "Paired VH:VL Antibody Repertoire Analysis", "Rep-Seq", "single cell immune repertoire sequencing", "10x immune profiling", "10x Ig enrichment", "10x TCR enrichment", "TCR Chain Paring", "MC-Hi-C", "CUT&RUN", "DNA-seq", "ATAC-seq", "Fast-ATAC", "scATAC-seq", "10x multiome", "10x scATAC-seq", "scATAC-seq (Microfluidics)", "scATAC-seq (cell index)", "massively parallel signature sequencing", "DNase Hi-C", "GAM", "IP-seq", "Bru-Seq", "CLIP-seq", "ChIP-exo", "ChIP-seq", "Chia-PET", "PLAC-seq", "RIP-seq", "Repli-Seq", "2-stage Repli-seq", "multi-stage Repli-seq", "eCLIP", "irCLIP", "MARGI", "MBD-seq", "MeDIP-seq", "Micro-C XL", "NAD-seq", "RNA-Seq", "SPRITE", "DNA SPRITE", "RNA-DNA SPRITE", "TSA-seq", "TraDIS sequencing", "Trac-Loop", "dilution HiC", "exome sequencing", "genotyping by high throughput sequencing", "in situ HiC", "methylation profiling by high throughput sequencing", "snmC-Seq2", "pooled clone sequencing", "sci-Hi-C", "single cell Hi-C", "sn-Hi-C", "spatial transcriptomics by high-throughput sequencing", "NanoString digital spatial profiling", "Slide-seq", "Slide-seqV2", "Visium Spatial Gene Expression", "transcription profiling by high throughput sequencing", "Quant-seq", "RNA-seq of coding RNA", "RNA-seq of coding RNA from single cells", "RNA-seq of non coding RNA", "RNA-seq of non coding RNA from single cells", "microRNA profiling by high throughput sequencing", "RNA-seq of total RNA", "random RNA-Seq across whole transcriptome", "single cell sequencing", "full length single cell RNA sequencing", "tag based single cell RNA sequencing", "single nucleus RNA sequencing", "full length single nucleus RNA sequencing", "tag based single nucleus RNA sequencing", "whole chromosome random sequencing", "16S metagenomic sequencing", "gRNA-seq", "shRNA-seq", "Capture-HiC", "scDNase-seq" ],
```

Comment:
Having the assay type and assay subtype attributes with specific CVs is OK for the first draft of the schemas. In the future, we should consider ways to take advantage of existing ontologies, which have already classified and structured all these terms. We should also consider ways to make it easy for users to provide a meaningful value. For example, users can provide the most specific term that describes their assay type, and we can infer more general classifications using ontology tools.

Reply:
Yes, for now I used CVs because we have not implemented automated ontology validations. As discussed we could also add already tips for the future like ``"meta:child_node_of": ...``.

Creating these CVs is tiresome, it's even worse when users need to suggest additions all the time, and we should just use ontologies hierarchies, definitions and descriptions. We could already dedicate time to: (1) reproduce what Biovalidator did (in-situ ontology validation); (2) find a way to download the current hierarchy and place that as a CV (saves time for the moment and shouldn't take that much time as the first option). 

Regarding the last part (inferring general classifications), it does take some time and effort on our side to infer them, and what I suggest is to also request for that general classification from the users. That way we won't be wondering if we chose correctly or not what they meant.